worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -135,6 +136,9 @@
     "fawe.error.input-parser-exception": "Invalid empty string instead of boolean.",
     "fawe.error.invalid-boolean": "Invalid boolean {0}",
     "fawe.error.schematic.not.found": "Schematic {0} not found.",
+    "fawe.error.parse.invalid-dangling-character": "Invalid dangling character {0}.",
+    "fawe.error.parse.unknown-mask": "Unknown mask: {0}, See: https://github.com/IntellectualSites/FastAsyncWorldEdit-Documentation/wiki/Masks",
+    "fawe.error.parse.unknown-transform": "Unknown transform: {0}, See: https://github.com/IntellectualSites/FastAsyncWorldEdit-Documentation/wiki/Transforms",
```

Comment:
Parse links within code. If we update them at any time for any reason, existing translations do not update. Putting this here for both, put that in a ClickEvent#openUrl. That is more convenient, else people cannot click links.